Cargo.toml

```diff
@@ -1,9 +1,22 @@
 [package]
 name = "saito-rust"
 version = "0.1.0"
-authors = ["Clayton Rabenda <clay@saito.tech>"]
+authors = ["Clayton Rabenda <clay@saito.tech>", "Stephe Peterkins <stephen@saito.tech>"]
 edition = "2018"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+tokio = { version = "1", features = ["full"] }
+chrono = "0.4"
+sha2 = "0.8"
+digest = '0.8'
+rand = "0.4"
+serde = { version = "1.0", features = ["derive"] }
+serde_bytes = "0.11"
+secp256k1 = { version = '0.12.2', features = ["rand", "serde"] }
+base58 = '0.1.0'
+merkle = '1.10.0'
+bincode = "1.1.4"
+ring = "^0.13.0"
+data-encoding = '2.1.2'
```

Comment:
Tokio is up to date, but the rest should be checked. I kept them as-is because I had some versioning trouble the first time around, and I know that this configuration works

Reply:
Some with `"` some with `'` some with `^` some with fixed version?  
- Does prettier work with toml files (to keep one code style?)
- Is cargo working with a lock file (in that case `^` makes sense in general, no?)?
- Should we have a look at deps-robots like https://dependabot.com/rust/ ?